apk-generator/v2/app/static/js/main.js

```diff
@@ -117,6 +123,46 @@ function hideProgress() {
     $fileProgressHolder.hide();
 }
 
+function connectSocketLogs() {
+  var socketLogs = io.connect(location.protocol + '//' + document.domain + ':' +
+      location.port + '/' + identifier + '/logs');
+
+  socketLogs.on('message', function(message) {
+	if (message === "SUCCESS") {
```

Comment:
This. The code inside this wouldnt' even be executed right ? How will download url be equal to `SUCCESS`